sqlalchemy/rpc_query.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+"""
+Four methods to interact with database.
+get(), put(), post(), delete()
+"""
+from models import engine, Data
+from sqlalchemy.orm import sessionmaker
+
+Session = sessionmaker(bind=engine)
+
+
+def get():
+    """
+    The get() will return the all the value from database.
+    """
+    session = Session()
+    _list = []
+    values = session.query(Data).all()
+    for value in values:
+        data = {
+            "id": value.id,
+            "title": value.title,
+            "body": value.body
+        }
+        _list.append(data)
+
+    _data = {"object": _list}
+    session.close()
+    return _data
+
+
+def post(title, body):
+    """
+        The post() will add a new record in database.
+    """
+    session = Session()
+
+    data = Data()
+    data.title = title
+    data.body = body
+    session.add(data)
+    session.commit()
+
+    session.close()
+    return 'Success'
```

Comment:
These methods should return boolean instead of string. So that the callers can then decide how to handle the results.